locale/en_US.md

```diff
@@ -1,57 +1,57 @@
-Is this punishment?
-Is it my ordeal?
-Won't the vigil I've been holding for years ever end?
-I'm like a little snowflake
-Melting back in your palm, come back baby
-
-If I see your tears
-My wings melt
-I can't fly to you in your dreams
-Eternity started with you
-In my little world
-Remember when you're gone I'm halfway done
-
-I'm in the deserts, I'm burning
-I'm at the pole, I'm cold
-I'm taking the punishment, please come back
-I'm reaching out, I can't hold
-I miss you, I'm crying
-I don't understand are you banned, please come back
-
-Even though you don't like me, I miss your voice
-Even if I say go, you don't go
-Is this longing or suffering that I have suffered for years?
-Are you haram for me if I knew
-
-Even though you don't like me, I miss your voice
-Even if I say go, you don't go
-Is this longing or suffering that I have suffered for years?
-Are you haram for me if I knew
-
-My baby, my ghost lover
-My baby, my ghost lover
-
-Ghost lover
-
-I'm in the deserts, I'm burning
-I'm at the pole, I'm cold
-I'm taking the punishment, please come back
-I'm reaching out, I can't hold
-I miss you, I'm crying
-I don't understand are you banned, please come back
-
-Even though you don't like me, I miss your voice
-Even if I say go, you don't go
-Is this longing or suffering that I have suffered for years?
-Are you haram for me if I knew
-
-Even though you don't like me, I miss your voice
-Even if I say go, you don't go
-Is this longing or suffering that I have suffered for years?
-Are you haram for me if I knew
-
-My baby, my ghost lover
-My baby, my ghost lover
-
-Ghost lover
+Is this punishment?  
+Is it my ordeal?  
+Won't the vigil I've been holding for years ever end?  
+I'm like a little snowflake  
+Melting back in your palm, come back baby  
```

Comment:
```suggestion
Melting back in your palm, come back baby
```

Don't add these spaces to the *last* line of each stanza. Every other line needs them, but the last lines should not. Otherwise the blank line gets parsed as part of the paragraph instead of being a paragraph break. The two spaces are a break but paragraph continuation.